llmspec/llmspec.py

```diff
@@ -65,6 +67,30 @@ def to_model(self, model: str = "ChatGLM"):
                 top_p=self.top_p,
                 temperature=self.temperature,
             )
+        elif model.lower() == "moss":
+            prompt = []
+            for message in self.messages:
+                if message.role == Role.USER:
+                    message_prefix = "<|Human|>"
+                elif message.role == Role.ASSISTANT:
+                    message_prefix = "<|MOSS|>"
+                else:
+                    message_prefix = ""
+                prompt.append(f"{message_prefix} {message.content}")
+            prompt = "\n".join(prompt) + "\n<|MOSS|>:"
```

Comment:
Seems we'd better split it into `get_prompt()` and `get_inference_args()`?

Reply:
That'd be nice! Working on it.

Reply:
Where should we put get_prompt()?